tests/Feature/JwtAuthorizationTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+
+namespace Marketredesign\MrdAuth0Laravel\Tests\Feature;
+
+use Closure;
+use Illuminate\Foundation\Application;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Route;
+use Illuminate\Testing\TestResponse;
+use Marketredesign\MrdAuth0Laravel\Tests\TestCase;
+
+class JwtAuthorizationTest extends TestCase
+{
+    private const ROUTE_URI = 'test';
```

Comment:
The userID, route_URI and response of the testing-route are all 'test', making it confusing to distinguish plus can go wrong if they return the wrong one which also happens to be 'test'. I suggest using 'test_route' and such.

Reply:
Updated as suggested